recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
IMO this should just create ``/opt/timesync/`` as the git checkout will create the source directory. Can you see if that works?